adam-pipeline/README.md

```diff
@@ -0,0 +1,67 @@
+# Computational Genomics Lab, Genomics Institute, UC Santa Cruz
+### Running the ADAM preprocessing pipeline container
+
+This guide will walk through running the ADAM preprocessing pipeline. If you
+find any errors or corrections please feel free to make a pull request against
+the [cgl-docker-lib](https://github.com/BD2KGenomics/cgl-docker-lib) repository,
+where this docker image is maintained. Feedback of any kind is appreciated.
+
+## Overview
+
+This container runs the [ADAM preprocessing
+pipeline](https://github.com/BD2KGenomics/toil-scripts/tree/master/src/toil_scripts/adam_pipeline),
+which is built using [Toil](https://github.com/BD2KGenomics/toil), a
+high-performance workflow execution system. [ADAM](https://github.com/bigdatagenomics/adam)
+is a parallel system for processing genomic data that is built on top of [Apache
+Spark](https://spark.apache.org). Although ADAM and Apache Spark are designed
+to be run as a distributed system, this container is designed to run on a
+single node. If you would like to run ADAM on a Spark cluster, look at the [ADAM
+preprocessing workflow in
+toil-scripts](https://github.com/BD2KGenomics/toil-scripts/tree/master/src/toil_scripts/adam_pipeline).
+
+This pipeline expects one input, which is aligned reads in SAM/BAM/ADAM format.
+Additionally, we expect a path to a known sites file. This is a file that
+describes positions where variants are known to occur, and is used to mask
+out sites during base quality score recalibration. Typically, a dbSNP VCF file
+is used. This pipeline requires a run environment with at least 10G of memory
+to run BQSR with a known sites file from dbSNP.
+
+This pipeline requires a host with Docker 1.9.0 installed. Other versions of
+Docker will soon be supported.
+
+## Genomic tool containers
```

Comment:
I would remove this section. The commit SHA below is bound to get stale, rendering the WAR ineffective. Can we automate the WAR?